.github/workflows/integration-tests.yml

```diff
@@ -3,7 +3,8 @@ on:
   merge_group:
   pull_request:
   schedule:
-    - cron: "0 0 * * *"
+    # - cron: "0 0 * * *"
+    - cron: "0 * * * *" # every hour while we debug for flakes
```

Comment:
@kalverra please add in the comment that this is temporary solution and add `TODO` to remove if after XXX